core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3146,12 +3146,16 @@ public void setAttribute(AlluxioURI path, SetAttributeOptions options)
       }
     }
     String commandName;
+    boolean checkWritableMountPoint = false;
     if (options.getOwner() != null) {
       commandName = "chown";
+      checkWritableMountPoint = true;
     } else if (options.getGroup() != null) {
       commandName = "chgrp";
-    } else if (options.getMode() != null) {
```

Comment:
this is a bug that `options.getMode()` will never be `null` as the default value is already `Constants.INVALID_MODE`.
So it will show an wrong command name as `chmod` if it is really Pin or Setttl